src/main/java/com/novoda/notils/exception/DeveloperError.java

```diff
@@ -12,7 +12,12 @@ public DeveloperError(String detailMessage) {
         super(detailMessage);
     }
 
+    public DeveloperError(String messageTemplate, String... args) {
```

Comment:
OOoo0o0oooOOOO `Formattable` !!!! cc @Electryc for our "when to use `toString`" discussions :)